bin/pycbc_live

```diff
@@ -457,7 +458,8 @@ class LiveEventManager(object):
 
         logging.info('computing followup data for coinc')
         coinc_ifos = coinc_results['foreground/type'].split('-')
-        followup_ifos = list(set(ifos) - set(coinc_ifos))
+        followup_ifos = set(ifos) - set(coinc_ifos)
+        followup_ifos = list(followup_ifos | set(self.skymap_only_ifos))
```

Comment:
I think this also needs to be done in the singles case at around line 547

Reply:
I was just about to write that!

Reply:
Yes, it was done probably [here](https://github.com/gwastro/pycbc/pull/4346/commits/91ee29ff2c20fa18ee88f01fd01ad5b4fca883e2), but seemed to have failed tests